package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@vheemstra/avifenc-bin",
   "version": "2.1.1",
```

Comment:
Version should be 3.0.0, since we're upgrading to libavif 1.3.0, which is a major version change.